### ISSUE TITLE
Pause and resume face detection?

### ISSUE THREAD

**Issue by molo32** [iperov/DeepFaceLab#660]
how can i pause and resume face detection
in a lot of frames like> 100,000 frames
?
If I run detect_face a second time it says that the content of "aligned" will be removed.

also the detector is very slow 2.70it / s

**Comment by blanuk** [comment]
speed is normal. 
i can manually export the already done frames. copy aligned and aligned_debug folder out and move the already done frames in a subfolder. at the ende, merge them together

**Comment by wuffenberg** [comment]
An option to pause is to use process explorer and suspend the python.exe process.

Anyway, in 100.000 frames, which equals to an hour+ long video, it makes sense to manually weed out the scenes / pics where the target person is not visible, thus reducing the # of frames to detect to less than 30.000. 
Also running 2 GPUs in parallel significantly speeds up detection.

**Comment by squalsy** [comment]
It is possible to run it in smaller batches. Move or rename the aligned folder for each time you run face detect, then when you are finished add all the aligned images to your aligned folder

**Comment by ghost** [comment]
Issue solved, please close it.
